composables/use-text-to-speech.ts

```diff
@@ -212,7 +212,11 @@ export function useTextToSpeech(options: TTSOptions = {}) {
 
   async function startTextToSpeech() {
     if (!user.value?.isLikerPlus) {
-      subscription.openPaywallModal()
+      subscription.openPaywallModal({
+        utmSource: 'epub_reader',
+        utmCampaign: nftClassId,
+        utmMedium: 'tts',
+      })
```

Comment:
I suggest to merge `utmOverrides` with `props` anyway, it is more intuitive to use, no need to an pass empty object